UnityProject/Assets/Scripts/Input System/CablePlacementVisualisation.cs

```diff
@@ -61,6 +61,10 @@ public class CablePlacementVisualisation : MonoBehaviour
 	/// Vector used to check distance between connections
 	/// </summary>
 	private Vector2Int endPointVector;
+	/// <summary>
+	/// The target tile the cable will be placed on.
+	/// </summary>
+	private GameObject target;
```

Comment:
This is getting the CablePlacementVisualisation itself since, since tiles are not game objects?

Reply:
I assumed it meant the tile. 
![image](https://user-images.githubusercontent.com/56727168/84509009-10dcf780-acbb-11ea-9173-b6f28bc0d3de.png)

Reply:
could be that it makes a game object that overlays the tile?

Reply:
![image](https://user-images.githubusercontent.com/56727168/84513700-bbf0af80-acc1-11ea-95fa-07644d0c7e47.png)

Reply:
humm, Sorry I'm not too good reading images xD

Reply:
```/// <summary>
	/// Creates a CableApply interaction performed by the local player targeting the specified object.
	/// </summary>
	/// <param name="targetObject">object targeted by the interaction, null to target empty space</param>
	/// <param name="wireEndA">cable start connection</param>
	/// <param name="wireEndB">cable end connection</param>
	/// <param name="worldPositionTarget">position of target tile (world space)</param>
	/// <returns></returns>
	public static ConnectionApply ByLocalPlayer(GameObject targetObject, Connection wireEndA, Connection wireEndB, Vector2? targetVector)
	{
		if (PlayerManager.LocalPlayerScript.IsGhost) return Invalid;

		var targetVec = targetVector ?? Camera.main.ScreenToWorldPoint(CommonInput.mousePosition) -
						PlayerManager.LocalPlayer.transform.position;

		return new ConnectionApply(
			PlayerManager.LocalPlayer,
			UIManager.Hands.CurrentSlot.ItemObject,
			targetObject,
			wireEndA,
			wireEndB,
			targetVec,
			UIManager.Instance.hands.CurrentSlot.ItemSlot,
			UIManager.CurrentIntent
		);
	}```